src/sage/categories/finite_groups.py

```diff
@@ -101,12 +101,19 @@ def cardinality(self):
                 sage: G.cardinality()
                 384
             """
+
+            # Check whether the object has _deg attribute or not
```

Comment:
this line is not needed - it's clear from the code

Reply:
Done ! I have removed all unnecessary comments.